fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2704,6 +2704,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long mv_plan_cache_max_size = 1000;
 
+    /**
+     * Max cache key size during query lifecycle to avoid repeating computes.
+     */
+    @ConfField(mutable = true)
```

Comment:
write the comment in annotation, so that it could be displayed in `admin show config`